pkg/cmd/cmd.go

```diff
@@ -75,11 +76,41 @@ func NewJXCommand(f clients.Factory, in terminal.FileReader, out terminal.FileWr
 	err io.Writer, args []string) *cobra.Command {
 
 	configureViper()
+
+	// It is likely that the user has the completions for kubectl loaded, so reusing function from there if they exist
+	bashCompletionFunction := `
```

Comment:
There is a completion command in `pkg/cmd/completion.go` which generates the completions for whole commands. I am wondering if you could move this into that command.

Reply:
OK